swagger/v1/api_spec.json

```diff
@@ -3398,35 +3260,23 @@
             "example": [
               {
                 "id": "db3a7848-7308-4879-942a-c4a70ced400a",
-                "type": "npq_application",
+                "type": "participant",
```

Comment:
Do you mind undoing this change as it's overwriting the correct response with the incorrect "participant" responce, as this is an NPQ application endpoint? see https://cpd-ecf-review-3882-web.test.teacherservices.cloud/api-reference/reference-v1.html#api-v1-npq-applications-get-parameters

Reply:
Hi @cwrw tks for spotting this.. I've fixed the issue .. `MultipleNPQApplicationsResponse` & `NPQApplicationAttributes` files had the incorrect attributes. I hope it's all good.. tksss